command-utility-tool/src/main/java/com/utility/InputHandler.java

```diff
@@ -0,0 +1,108 @@
+package com.utility;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+/**
+ * This class handles user input and retry 3 times if input is invalid.
```

Comment:
Is the requirement that retry 3 times?

Reply:
I modified the code but forgot to change the description, right now, if the user inputs the wrong path or non-existing file name, the code will throw exceptions. If the user inputs the invalid word, will keep input until its valid, then do the process of looking for occurrence.